src/parsers/manifest/dash/common/get_clock_offset.ts

```diff
@@ -26,15 +26,23 @@ import log from "../../../../log";
  *
  * For example, a response of 1000 would mean that performance.now() is 1 second
  * behind the server's time.
+ *
+ * Sometimes, it may happen that the server is unreacheable for somes reasons,
+ * if we fall in this case, we will define an offset of 0.
  * @param {string} serverClock
  * @returns {number|undefined}
  */
 export default function getClockOffset(
```

Comment:
I find it strange to still call `getClockOffset` if the server clock could not have been fetched.
Shouldn't we just avoid to call it?